src/common/transformations/src/transformations/fp16_compression/mark_subgraphs_to_keep_in_mixed_precision.cpp

```diff
@@ -265,6 +267,33 @@ class MarkExp : public pass::MatcherPass {
         register_matcher(m, callback);
     }
 };
+
+class MarkRandomUniform : public pass::MatcherPass {
+public:
+    OPENVINO_MATCHER_PASS_RTTI("MarkRandomUniform");
+
+    MarkRandomUniform() {
+        MATCHER_SCOPE(MarkRandomUniform);
+        auto random_uniform_pattern = pattern::wrap_type<ov::op::v8::RandomUniform>();
+
+        matcher_pass_callback callback = [=](pattern::Matcher& m) {
+            const auto& node = m.get_match_root();
+            if (!node)
+                return false;
+
+            disable_fp16_compression(node);
+            for (const auto& output : node->outputs()) {
+                for (const auto& out_inputs : output.get_target_inputs()) {
+                    mark_as_precision_sensitive(out_inputs);
+                }
+            }
+            return true;
```

Comment:
usually we "return false" if the graph structure is not changed.

Reply:
but as I can see MarkExp also returns true.
probably @pavel-esir might comment it

Reply:
Done, thanks!